js/deer-render.js

```
@@ -278,7 +284,12 @@ export default class DeerRender {
                     if(e.detail.target.closest(DEER.VIEW+","+DEER.FORM).getAttribute("id")===listensTo) elem.setAttribute(DEER.ID,e.detail.target.closest('['+DEER.ID+']').getAttribute(DEER.ID))
                 } catch (err) {}
             })
-            window[listensTo].addEventListener("click", e => UTILS.broadcast(e,DEER.EVENTS.CLICKED,elem))
+            if(window[listensTo] !== undefined){
```

try {} this instead, log an error and silent fail.